ctapipe/tools/train_disp_reconstructor.py

```diff
@@ -0,0 +1,196 @@
+import astropy.units as u
+import numpy as np
+
+from ctapipe.core import Tool
+from ctapipe.core.traits import Bool, Int, Path, TraitError, flag
+from ctapipe.io import TableLoader
+from ctapipe.reco import CrossValidator, DispReconstructor
+from ctapipe.reco.preprocessing import check_valid_rows, horizontal_to_telescope
+
+
+class TrainDispReconstructor(Tool):
+    """
+    Tool to train a `~ctapipe.reco.DispReconstructor` on dl1b/dl2 data.
+
+    The tool first performs a cross validation to give an initial estimate
+    on the quality of the estimation and then finally trains two models
+    (|disp| and sign(disp)) per telescope type on the full dataset.
+    """
+
+    name = "ctapipe-train-disp-reconstructor"
+    description = __doc__
+
+    examples = """
+    ctapipe-train-disp-reconstructor \\
+        --config train_disp_reconstructor.yaml \\
+        --input gamma.dl2.h5 \\
+        --output disp_models.pkl
+    """
+
+    output_path = Path(
+        default_value=None,
+        allow_none=False,
+        directory_ok=False,
+        help=(
+            "Ouput path for the trained reconstructor."
+            " At the moment, pickle is the only supported format.",
+        ),
+    ).tag(config=True)
+
+    n_events = Int(
+        default_value=None,
+        allow_none=True,
+        help=(
+            "Number of events for training the models."
+            " If not give, all available events will be used."
+        ),
+    ).tag(config=True)
+
+    overwrite = Bool(help="overwrite existing output files").tag(config=True)
+
+    random_seed = Int(
+        default_value=0, help="Random seed for sampling and cross validation"
+    ).tag(config=True)
+
+    project_disp = Bool(
+        default_value=False,
+        help="Project true source position on main shower axis for true |disp| calculation",
```

Comment:
Please clarify what the differences are between True and False here

Reply:
Done.